package.json

```diff
@@ -71,9 +71,12 @@
     }
   },
   "dependencies": {
+    "@remix-project/remixd": "^0.3.2",
```

Comment:
latest version of remixd is `0.3.5`

Reply:
Would you not like to update the version in this PR?

Reply:
yes I will once we've deployed the last remixd

Reply:
(which is about to be published)